larky/src/test/resources/vgs_tests/http/test_default_request.star

```diff
@@ -104,10 +104,28 @@ def _test_request_get_full_url():
     asserts.assert_that(request.full_url).is_equal_to(url)
 
 
-def _test_request_get_relative_url():
-    relative_url = '/path;parameters?query=argument'
+def _test_request_get_url():
+    url = 'http://netloc/path;parameters?query=argument#fragment'
```

Comment:
can you test with duplicate query parameters?

Reply:
done https://github.com/verygoodsecurity/starlarky/pull/243/commits/d0c8694d53736d956f867ecb0f525f649cac2baa !